Example/Example/ViewController.swift

```diff
@@ -7,11 +7,32 @@
 //
 
 import UIKit
+import Rover
+import os
 
 class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        let experienceFile = Bundle.main.path(forResource: "experience.json", ofType: nil)!
+        
+        let experienceFileURL: URL = URL(fileURLWithPath: experienceFile)
+        os_log("Experience file path: %@", experienceFileURL.absoluteString)
+        let experienceJson = try! Data(contentsOf: experienceFileURL)
+        
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .formatted(DateFormatter.rfc3339)
+        
+        let experience = try! decoder.decode(Experience.self, from: experienceJson)
+        
+        
+        let rvc = RoverViewController.init()
+        rvc.loadExperience(experience: experience)
+        
+        self.present(rvc, animated: false)
+    }
```

Comment:
this is in here temporarily during the Polls project. I will be reverting it before release.